Packages/ClientRuntime/Sources/PrimitiveTypeExtensions/String+Extensions.swift

```diff
@@ -113,3 +113,14 @@ extension String {
         return ByteBuffer(data: self.data(using: .utf8) ?? Data())
     }
 }
+
+public extension String {
+    func stripFirstMatching(prefixes: [String]) -> String {
+        for prefixToStrip in prefixes {
```

Comment:
```suggestion
        for prefixToStrip in prefixes where self.hasPrefix(prefixToStrip) {
```
could be shortened to the above :)

Reply:
I didn't think of using `where` there, but semantically using `where` seems like it would imply returning an array (since we are starting with an array), and I'm only expecting one to case to hit.  I prefer the `if` statement over the `where`.  I'm open to hear other suggestions, but for now i'm going to leave it.